source/login_form.php

```diff
@@ -45,10 +46,10 @@
             <?php endif ?>
         </p>
         <p>
-            <input type="submit" value="ログイン">
+            <input type="submit" value="ログイン" class="btn btn-primary">
```

Comment:
テストで挿入しているclassがありますので削除して下さい。
```suggestion
            <input type="submit" value="ログイン">
```

Reply:
対応しました。